pages/account/how-to/use-2fa.mdx

```diff
@@ -32,8 +32,10 @@ Download the app of your choice and install it onto your smartphone.
 
 ## How to enable MFA
 
-1. Access the [Security](https://console.scaleway.com/account/security) tab of your **User Account** page.
-  Alternatively, click your Organization name on the top-right corner of the console navigation menu, click **Profile**, then **Security**.
+1. Click your Organization name on the top-right corner of the console navigation menu, click **Profile**, then **Security**.
+    <Message type="important">
+      If you are logged in as an [IAM member](/iam/concepts/#member), Click **Profile**, then **Credentials** and scroll down to the **Multifactor authentication** section.
```

Comment:
I thought we always wrote "Member" with a capital M?

Reply:
we do :) je fix